numpy/ma/core.pyi

```diff
@@ -666,7 +669,57 @@ class MaskedArray(ndarray[_ShapeType_co, _DType_co]):
         keepdims: bool = False,
     ) -> _ArrayT: ...
 
-    #
+    # Keep in-sync with np.ma.take
+    @overload
+    def take(  # type: ignore[overload-overlap]
+        self: _MaskedArray[_SCT],
+        indices: _IntLike_co,
+        axis: None = None,
+        out: None = None,
+        mode: _ModeKind = 'raise'
+    ) -> _SCT: ...
+    @overload
+    def take(
+        self,
+        indices: _IntLike_co,
+        axis: SupportsIndex | None = None,
+        out: None = None,
+        mode: _ModeKind = 'raise',
+    ) -> Any: ...
```

Comment:
The overload above always gets matched, so unless I'm missing something, this one can be removed.

```suggestion
```

Reply:
ah, thanks! I was under the impression that some masked arrays could be of dtypes that might not be possible to match against `MaskedArray[_SCT]`, but I should have asked

have removed 👍